.github/workflows/add_comment.yml

```diff
@@ -0,0 +1,89 @@
+name: Add "bors r+" comment
+
+on:
+  pull_request:
+    types: [labeled]
```

Comment:
To use this action we have to approve first and then label (whereas with mergify we could label and then approve). We can always un-label and re-label if we happen to do it in the other order.

Reply:
It also doesn't work if there was a `bors r-` and `bors r+` comment, or if the author of the PR pushed another commit and you need to re-review by writing `bors r+` a second time.

I'm not sure the label+approval->comment workflow is a good idea.  It hides too much of bors.  You'll certainly need to write `bors r+` manually at some point.  So I think people should get used to the comments.

Reply:
I agree. I wouldn't mind removing this workflow entirely, but it might help while we switch over.

Reply:
I'm feeling more and more inclined to remove this workflow. We seem to label and then approve a lot more than I thought we did, and I think having something that's so unreliable will be more annoying than just switching over.